Source/Urho3D/Utility/GLTFImporter.cpp

```diff
@@ -2725,7 +2725,7 @@ class GLTFModelImporter : public NonCopyable
 
                 const auto colors = bufferReader_.ReadAccessorChecked<Vector3>(accessor);
                 for (unsigned i = 0; i < accessor.count; ++i)
-                    vertices[i].color_[semanticsIndex] = { colors[i], 1.0f };
+                    vertices[i].color_[semanticsIndex] = colors[i].ToVector4(1.0f);
```

Comment:
imho old version was more obvious what happens here.

Reply:
@eugeneko also thinks this should be rolled back. For shader args constructor is more appropriate.